### ISSUE TITLE
Console errors in dev mode

### ISSUE THREAD

**Issue by ingro** [flipace/lovli.js#4]
Hello I'm running lovli in dev mode and every two minutes it logs this error in the console:

```
Unhandled rejection ReqlOpFailedError: Table `lovli.todos` does not exist in:
r.table("todos").insert({
^^^^^^^^^^^^^^^^
    text: "Invest in shrimps."
})

    at Connection._processResponse (D:\projects\lovli.js\node_modules\rethinkdbdash\lib\connection.js:382:15)
    at Socket.<anonymous> (D:\projects\lovli.js\node_modules\rethinkdbdash\lib\connection.js:201:14)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)
```

For what I understand, in dev mode lovli is inserting every two minutes a random todo, and tries to delete them all every 10 minutes.

Problem is that the table created automatically by horizon is not called `todos`, but something like `todos_f75f4192870d` (I guess it's a random name), hence the errors.

**Comment by flipace** [comment]
Hi, yes lovli is going to add random todos automatically. 

You could either remove or adapt the offending code for now or wait until I figured out a solution. I think I'll have to get in touch with the horizon devs to fix this correctly.

**Comment by ingro** [comment]
No problem, that's not a big deal, just wanted to report it :)

**Comment by flipace** [comment]
fixed #10
